mongocrypt/build.gradle.kts

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import de.undercouch.gradle.tasks.download.Download
+
+buildscript {
+    repositories {
+        mavenCentral()
+        google()
+    }
+    dependencies {
+        "classpath"(group = "net.java.dev.jna", name = "jna", version = "5.11.0")
+    }
+}
+
+plugins {
+    // Needed to download libmongocrypt from s3.
+    id("de.undercouch.download") version "5.6.0"
+}
+
+group = "org.mongodb"
+base.archivesBaseName = "mongodb-crypt"
+description = "MongoDB client-side crypto support"
+ext.set("pomName", "MongoCrypt")
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    api(project(path = ":bson", configuration = "default"))
+    api("net.java.dev.jna:jna:5.11.0")
+
+    // Tests
+    testImplementation("org.junit.jupiter:junit-jupiter")
+    testRuntimeOnly("ch.qos.logback:logback-classic:1.2.13")
```

Comment:
Everywhere else this is `testImplementation`.  Does it need to be different here?

Reply:
I removed this line as per @rozza suggestion here: https://github.com/mongodb/mongo-java-driver/pull/1487#discussion_r1763156734